scipy/docker/requirements.txt

```diff
@@ -0,0 +1,20 @@
+#aerospike==2.1.2
```

Comment:
Why are these requirements here?

Reply:
They're a useful basis.  Shall I change to just numpy, scipy, scikit-learn?